### DISCUSSION TITLE
帮忙解决下专业版账户会员没有激活问题

### DISCUSSION THREAD

**Discussion by YongboShen** [getlantern/lantern-forum-cn#920] [Q&A]
![微信图片_20241003013900](https://github.com/user-attachments/assets/2d2162b3-5b46-4a2f-9c9b-72d0ae1d3ee5)

已经支付了会费，但是一直没有收到激活信息，邮箱 yongboshen@gmail.com

**Comment by MaxMKMax** [comment]
@YongboShen 您好. 谢谢您支持蓝灯. 已经帮您提交问题上去给负责人处理了.

**Reply by YongboShen** [reply]
尚未解决。。。邮件验证码一直没有收到

**Reply by YongboShen** [reply]
@Mike777Khoo Mike，能否帮我更新下进度，仍然没有办法升级成专业版，如果不行的话请退款，谢谢。

**Reply by lirui10093622** [reply]
@Mike777Khoo 我也是同样的问题，拜托处理一下。邮箱：lirui10093622@163.com

**Reply by LuLu120325** [reply]
请帮我处理退款，给个明确的回复，谢谢，邮箱：2647611833@qq.com

**Reply by wwei20240912** [reply]
@Mike777Khoo  我也是购买专业版本，根本无法登录，邮件收不到验证码，请退款，邮箱：94472127@qq.com
